### PR TITLE
Implemented round for scalars in pandas backend.

### PR DESCRIPTION
**Pull request by DiegoAlbertoTorres**

### CODE REVIEW

File: ibis/pandas/execution/generic.py [hunk 1]
```diff
@@ -278,6 +278,18 @@ def execute_cast_string_literal(op, data, type, **kwargs):
...
+@execute_node.register(
+    ops.Round,
+    scalar_types,
+    (six.integer_types, type(None))
+)
+def execute_round_scalars(op, data, places, **kwargs):
+    if places is None:
+        return np.around(data)
+    else:
+        return np.around(data, places)
```
[the remark this answers is not in the document]
Comment (DiegoAlbertoTorres) [reply ibis/pandas/execution/generic.py]: Nope:
```
>>> np.around(3.333, None)
Traceback (most recent call last):
  File "/home/diego/.local/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 52, in _wrapfunc
    return getattr(obj, method)(*args, **kwds)
AttributeError: 'float' object has no attribute 'round'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/diego/.local/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2837, in around
    return _wrapfunc(a, 'round', decimals=decimals, out=out)
  File "/home/diego/.local/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 62, in _wrapfunc
    return _wrapit(obj, method, *args, **kwds)
  File "/home/diego/.local/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 42, in _wrapit
    result = getattr(asarray(obj), method)(*args, **kwds)
TypeError: an integer is required (got type NoneType)
```

[the remark this answers is not in the document]
Comment (cpcloud) [reply ibis/pandas/execution/generic.py]: cool

**Comment by cpcloud** [comment]
@DiegoAlbertoTorres i'll merge once the appveyor build passes

**Comment by DiegoAlbertoTorres** [comment]
:confetti_ball: :tada:

**Comment by cpcloud** [comment]
@DiegoAlbertoTorres I'll rebase and merge this.

**Comment by cpcloud** [comment]
Merged this in 8b2f2be

**Pull request closed without merging**